src/main/haskell/kore/src/Kore/Builtin/Builtin.hs

```diff
@@ -123,83 +125,56 @@ type Function = ApplicationFunctionEvaluator Object
 
 type HookedSortDescription = SortDescription Object Domain.Builtin
 
+-- | @MonadVerify@ type constraint for monadic verification actions
+type MonadVerify m = MonadError (Error VerifyError) m
+
 -- | Verify a sort declaration.
-type SortDeclVerifier =
+type SortDeclVerifier m =
```

Comment:
I parameterized the DomainValueVerifiers because the PatternVerifier was parameterized, and since the other verifiers were of the same concrete type as the one DomainValueVerifiers were instantiated with, I figured it would be a good idea to have them parameterized as well.  But since DomainValueVerifiers can be concrete I could just make them all concrete.